axelar-chains-config/info/devnet-amplifier.json

```
@@ -1494,6 +1494,8 @@
     "grpc": "devnet-amplifier.axelar.dev:9090",
     "tokenSymbol": "AMPLIFIER",
     "gasPrice": "0.00005uamplifier",
-    "gasLimit": "auto"
+    "gasLimit": "auto",
+    "depositValue": "100000000",
```

```suggestion
    "govProposalDepositAmount": "100000000",
```
Use a more meaningful name

---

Update submit proposal script to read this value and make --deposit optional